### PR TITLE
Video talks

### PR DESCRIPTION
**Pull request by JoshMacSween**
Place for Video archive

### CODE REVIEW

File: app/controllers/pages_controller.rb [hunk 1]
```diff
@@ -16,4 +16,7 @@ def slack
 
   def resources
   end
+
+  def videos
+  end
```
Comment (alanvardy) [inline app/controllers/pages_controller.rb]: We don't need this, lets just use videos#index

File: app/assets/stylesheets/videos.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Videos controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
```
Comment (alanvardy) [inline app/assets/stylesheets/videos.scss]: These extra scss files screw things up. Please delete the new ones. We will be only using application.scss

File: app/assets/stylesheets/scaffolds.scss [hunk 3]
```diff
@@ -0,0 +1,84 @@
+body {
+  background-color: #fff;
+  color: #333;
+  margin: 33px;
+  font-family: verdana, arial, helvetica, sans-serif;
+  font-size: 13px;
+  line-height: 18px;
+}
```
Comment (alanvardy) [inline app/assets/stylesheets/scaffolds.scss]: These extra scss files screw things up. Please delete the new ones. We will be only using application.scss

File: config/routes.rb [hunk 4]
```diff
@@ -80,6 +89,7 @@
   get '/opportunities', to: 'pages#opportunities'
   get '/slack', to: 'pages#slack'
   get '/resources', to: 'pages#resources'
+  get '/videos', to: 'pages#video'
```
Comment (alanvardy) [inline config/routes.rb]: No need for this either

File: test/controllers/videos_controller_test.rb [hunk 5]
```diff
@@ -0,0 +1,48 @@
+require 'test_helper'
+
+class VideosControllerTest < ActionDispatch::IntegrationTest
+  setup do
+    @video = videos(:one)
```
Comment (alanvardy) [inline test/controllers/videos_controller_test.rb]: This syntax is wrong, which is why the tests are failing. Can you look at the other controller tests and figure out the difference?

We need to load the @video from factories.rb

File: config/routes.rb (hunk 4 above)
Comment (JoshMacSween) [reply config/routes.rb]: I don't understand why we didn't need this.

**Pull request merged**